addons/cookoff/initSettings.sqf

```diff
@@ -18,6 +18,16 @@
     false // Needs mission restart
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(destroyVehicleAfterCookoff), "CHECKBOX",
+    [LSTRING(destroyVehicleAfterCookoff_name), LSTRING(destroyVehicleAfterCookoff_tooltip)],
+    LSTRING(category_displayName),
+    true, // default value
```

Comment:
Default false. Intended behaviour is for them to remain there not blown up

Reply:
Changed it 👍

Reply:
> imo having tile default as false is a mistake.
> 
> Not having vehicles die breaks any currently existing script that counts vehicle destruction, this is a huge change from expected behaviour and has been one of the main complaints I've seen people have about this component.

Well I agree with this, otherwise I wouldn't have made this PR but I can't call myself an authority here. @TheCandianVendingMachine How do we resolve this?

Reply:
I agree with @Drofseh, the current default behaviour is a big change from vanilla game default behaviour and I believe it should be opt-in instead of opt-out.

Reply:
asked @acemod/maintainers for opinions.

Reply:
Maintainers have agreed the setting should remain off by default so the system works as it was designed, however, due to a big change from vanilla behaviour we want Advanced Vehicle Damage disabled by default.

Disabling of the system by default should be done in separate PR so it will be reflected in changelog.

Reply:
Can't get why after cookoff and full burnout vehicle should be not wrecked and even repairable?